treadmillbehaviordatainterface.py

```diff
@@ -0,0 +1,51 @@
+"""Authors: Cesar Echavarria and Ben Dichter."""
```

Comment:
```suggestion
"""Authors: Cesar Echavarria."""
```

no need to list me as an author for files you write on our own.